build.py

```diff
@@ -50,8 +50,8 @@
 # Map from container version to corresponding component versions
 # container-version -> (ort version, ort openvino version)
 CONTAINER_VERSION_MAP = {
-    '20.08': ('1.4.0', '2020.2'),
-    '20.09': ('1.4.0', '2020.2')
+    '20.08': ('1.5.1', '2020.4'),
+    '20.09': ('1.5.1', '2020.4')
```

Comment:
We didn't change what was used in 20.08 and 20.09 so don't change these. Instead add an entry for 20.10 with the new onnx versions. Perhaps this is just for the CI run...

Reply:
The issue is that there is mismatch between the dev version and the container version, i..e. this change targets on 20.10 but we are using 20.09 container. I think for now, we should change it for CI run, but then correct it on the release branch. 

In addition to that, the change in 20.08 is because current master is still using 20.08 due to the late 20.09 release.. We can change it once the master is updated to use 20.09 container